tests/pyfunc/test_model_export_with_class_and_artifacts.py

```diff
@@ -830,6 +830,23 @@ def test_log_model_with_unsupported_argument_combinations_throws_exception():
     assert "Either `loader_module` or `python_model` must be specified" in str(exc_info)
 
 
+@pytest.mark.large
+def test_repr_can_be_called_withtout_run_id_or_artifact_path():
+    model_meta = Model(
+        artifact_path=None,
+        run_id=None,
+        flavors={"python_function": {"loader_module": "someFlavour"}},
+    )
+
+    class TestModel(object):
+        def predict(self, model_input):
+            return model_input
+
+    model_impl = TestModel()
+
+    assert "flavor: someFlavour" in str(mlflow.pyfunc.PyFuncModel(model_meta, model_impl))
```

Comment:
```suggestion
    assert "flavor: someFlavour" in mlflow.pyfunc.PyFuncModel(model_meta, model_impl).__repr__()
```

Can we use `__repr__` instead?

Reply:
OK. Done.